### ISSUE TITLE
Classes with names int, string, float, bool... are forbidden in PHP7

### ISSUE THREAD

**Issue by wuestkamp** [doctrine/phpcr-odm#788]
I know there are some other closed issues about this, but when I use PHPStorms Language Level detection I'm getting the following errors:

```
Language Level
Float.php
Classes with names int, string, float, bool, true, false, and null are forbidden in PHP 7
Int.php
Classes with names int, string, float, bool, true, false, and null are forbidden in PHP 7
String.php
Classes with names int, string, float, bool, true, false, and null are forbidden in PHP 7
String.php
Classes with names int, string, float, bool, true, false, and null are forbidden in PHP 7
```

Files mentioned are in `vendor/doctrine/phpcr-odm/lib/Doctrine/ODM/PHPCR/Mapping/Annotations`. Can this be ignored? There is no error during runtime in our application.

Thanks,
Kim

**Comment by dbu** [comment]
are you using version 1.x of this library? i don't see those classes in the current (version 2) master: https://github.com/doctrine/phpcr-odm/tree/master/lib/Doctrine/ODM/PHPCR/Mapping/Annotations

**Comment by wuestkamp** [comment]
Yes using version 1.4.4 because there is no 2.x release yet. Though 1.4.4 should be ready for PHP ^7.0 according to its composer.json O:) It might be that those files are still kept to ensure PHP 5 compatibility in 1.x but aren't actually used with PHP 7.

**Comment by dbu** [comment]
yes, i think we never load them in php 7. and we had to keep them for BC because documents may still be using the old annotations.
